modules/local/purgedups/calcuts.nf

```diff
@@ -0,0 +1,32 @@
+process PURGEDUPS_CALCUTS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::purge_dups=1.2.6" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/purge_dups:1.2.6--h7132678_0':
+        'quay.io/biocontainers/purge_dups:1.2.6--h7132678_0' }"
+
+    input:
+    tuple val(meta), path(stat)
+
+    output:
+    tuple val(meta), path("*.cutoffs")    , emit: cutoff
+    tuple val(meta), path("*.calcuts.log"), emit: log
+    path "versions.yml"                   , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    calcuts $args $stat > ${prefix}.cutoffs 2> ${prefix}.calcuts.log
```

Comment:
I had a feeling this would be a possibility. Thanks for confirming it. I'll definitely try incorporate this.

Reply:
Is the `-g` option an internal modification? It's not part of the listed options.
```
# calcuts -h

Usage: calcuts  [<options>] <STAT> ...
Options:
         -f    FLOAT    minimum depth count fraction to maximum depth count [.1]
         -l    INT      lower bound for read depth
         -m    INT      transition between haploid and diploid
         -u    INT      upper bound for read depth
         -d             treat as haploid assembly or diploid assembly, 1: haploid, others: diploid [0]
         -h             help
```

Reply:
Sorry, I've just checked that it can be implemented as following: the `model.txt` file can be parsed inside of the pipeline and the thresholds can be further introduced into `calcuts` by leveraging `-m` and `-u` options.